docs/pages/installation.mdx

```diff
@@ -48,55 +48,391 @@ supports most features on Windows 10 and later.*
+  manually.
+
+If you are running a self-hosted Teleport Enterprise cluster, read our guidance
+in [Deploying a High-Availability Teleport
+Cluster](./deploy-a-cluster/introduction.mdx), provides full requirements and
+Terraform modules for deploying Teleport Enterprise.
+
+The instructions in this section are intended for:
+- Linux desktop users.
+- Operators who want to incorporate Teleport installation commands into custom
+  scripts when the approaches listed above are not suitable.
+
+### One-line installation script
+
+You can run a one-line command to install Teleport binaries on a Linux server:
 
-### Feature support
+```code
+$ EDITION="oss"
+$ VERSION="(=teleport.version=)"
+$ curl https://goteleport.com/static/install.sh | bash -s ${VERSION?} ${EDITION?} 
+```
 
-Some Teleport features have additional requirements:
+The command takes the Teleport version and edition to install, then uses
+information about the operating system where it runs to choose a package manager
+and install Teleport.
 
-| Feature                                                                        | Requirement   | Debian                   | Ubuntu   | CentOS/RHEL | Amazon Linux           | SLES           |
-|--------------------------------------------------------------------------------|---------------|--------------------------|----------|-------------|------------------------|----------------|
-| [Enhanced Session Recording](./server-access/guides/bpf-session-recording.mdx) | Kernel  v5.8+ | 11, or 10 with backports | 20.04.2+ | 9+          | 2 (post 11/2021), 2023 | 12 SP5, 15 SP5 |
-| Automatic Updates                                                              | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
-| Installation through apt/yum/zypper repos                                      | systemd-based | 9+                       | 16.04+   | 7+          | 2, 2023                | 12 SP5, 15 SP5 |
+Valid editions are:
 
-<Admonition type="note">
-`apt`, `yum`, and `zypper` repos don't expose packages for all distribution variants.
-When following installation instructions, you might need to replace `ID` with
-`ID_LIKE` to install packages of the closest supported distribution.
+| Value        | Edition                           |
+|--------------|-----------------------------------|
+| `cloud`      | Teleport Enterprise Cloud         |
+| `enterprise` | Teleport Enterprise (Self-Hosted) |
+| `oss`        | Teleport Community Edition        |
+| `team`       | Teleport Team                     |
 
-Currently supported distributions (and `ID`) are:
-- RHEL >= 7 (`rhel`)
-- CentOS >= 7 (`centos`)
-- Debian >= 9 (`debian`)
-- Ubuntu >= 16.04 (`ubuntu`)
-- Amazon Linux 2 and 2023 (`amzn`)
-- SLES >= 12 SP5, >= 15 SP5 (`sles`)
+If running Teleport Enterprise Cloud, Teleport agents must be running version
+(=cloud.version=) or one major version below.
```

Comment:
I think we should remove this remark because on Cloud we want everyone to be using automatic upgrades.

Reply:
Removing this paragraph and adding a more explicit recommendation for Cloud users to enable automatic agent upgrades.